core/state_transition.go

```diff
@@ -525,8 +528,11 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 	}, nil
 }
 
-// distributeTxFees calculates the amounts and recipients of transaction fees and credits the accounts.
-func (st *StateTransition) distributeTxFees() error {
+// creditTxFees calculates the amounts and recipients of transaction fees and credits the accounts.
+func (st *StateTransition) creditTxFees() error {
```

Comment:
Worth updating the function name in the log line [here](https://github.com/celo-org/celo-blockchain/pull/2258/files#diff-8c398ef0a79f97ba6d497a99247a815b3a51918de9e16954215b6073f907c92eR578) as well then.

Reply:
Done